tools/stmo.md

```diff
@@ -1,5 +1,276 @@
+Firefox organization's data sets. As the name and URL imply, effective use of
+this tool requires familiarity with the
+[SQL](https://en.wikipedia.org/wiki/SQL) query language, with which all of the
+tool's data extraction and analysis are performed.
+
+Concepts
+--------
+
+There are three building block from which analyses in STMO are constructed:
+queries, visualizations, and dashboards.
+
+#### Queries
+
+STMO's basic unit of analysis is the query. A query is a block of SQL code that
+extracts and (optionally) transforms data from a single data source. Queries
+can vary widely in complexity. Some queries are trivial one liners
+(e.g. 'SELECT * FROM tablename LIMIT 10'), while others are many pages long,
+small programs in their own right.
+
+The raw output from a query is tabular data, where each row is one set of
+return values for the query's output columns. A query can be run manually or it
+can be specified to have a refresh schedule, where it will execute
+automatically after a specified interval of time.
+
+#### Visualizations
+
+Tabular data is great, but rarely is a grid of values the best way to make
+sense of your data. Each query can be associated with multiple visualizations,
+each visualization rendering the extracted data in some more useful
+format. There are many visualization types, including charts (line, bar, area,
+pie, etc.), counters, maps, pivot tables, and more. Each visualization type
+provides a set of configuration parameters that allow you to specify how to map
+from the raw query output to the desired visualization. Some visualization types
+make demands of the query output; a map visualization requires each row to contain
+a longitude value and a latitude value, for instance.
+
+#### Dashboards
+
+A dashboard is a collection of visualizations, combined into a single visual
+presentation for convenient perusal. A dashboard is decoupled from any
+particular queries. While it is possible to include multiple visualizations
+from a single query in one dashboard, it is not required; users can add any
+visualizations they can access to the dashboards they create.
+
+Data Sources
+------------
+
+SQL provides the ability to extract and manipulate the data, but you won't get
+very far without having some familiarity with what data is actually available,
+and how that data is structured. Each query in STMO is associated with exactly
+one data source, and you have to know ahead of time which data source contains
+the information that you need. One of the most commonly used data sources is
+called *Athena* (referring to Amazon's [Athena](https://aws.amazon.com/athena/)
+query service, on which it is built), which contains most of the data that is
+obtained from telemetry pings received from Firefox clients. The *Athena*
+source is slowly replacing the *Presto* data source. *Presto* contains all of
+the data that's exposed via *Athena* and more, but returns query results much
```

Comment:
Is what true? I'm trying to say that Presto has everything but is slower, Athena doesn't have everything but is faster. Is that not what you're experiencing? Or maybe my wording is unclear here?